mainloop.cpp

```diff
@@ -114,7 +114,7 @@ int Mainloop::write_msg(Endpoint *e, const struct buffer *buf)
     return r;
 }
 
-void Mainloop::router_msg(Endpoint *sender, struct buffer *buf, int target_sysid)
+void Mainloop::router_msg(struct buffer *buf, int target_sysid, int sender_sysid)
```

Comment:
not from this commit, but this method should be called "route_msg"... i.e. a verb not a noun.